### PR TITLE
i#2488: fix links for memtrace and drltrace in README.md

### LINKED ISSUE #2488: Broken links in README.md
The links in section 'Existing DynamoRIO-based Tools' to the library tracing tool and memory tracing tool are currently broken.

### PR DESCRIPTION
**Pull request by fhahn**
Fixes #2488.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,8 +27,8 @@ Tools built on DynamoRIO include:
   [drcpusim](http://dynamorio.org/docs/page_drcpusim.html)
 - The "strace for Windows" tool [drstrace](http://drmemory.org/strace_for_windows.html)
 - The code coverage tool [drcov](http://dynamorio.org/docs/page_drcov.html)
-- The library tracing tool [drltrace](http://drmemory.org/docs/page_drltrace.html)
```
Comment (derekbruening) [inline README.md]: As you can see from the history, drltrace was recently moved to the drmemory repo and @mxmssh made this change to drmemory.org which is where the page will eventually be.  However the online docs have not yet been updated as they match release packages.  It's not clear when the next Dr. Memory release will be (too few people working on it...) so we'll have to decide what to do here.

If we had published nightly build packages (not yet added to Travis or Appveyor: #1967) we could point at the latest one of those?

Comment (fhahn) [inline README.md]: Ah I see. Would it make sense to set the link to https://github.com/DynamoRIO/drmemory/tree/master/drltrace until the drmemory.org link works?

Comment (derekbruening) [inline README.md]: @mxmssh -- does that work for you?  It is a step down from the html page but you could spice up that drltrace/README.md page separately.

Comment (mxmssh) [inline README.md]: Yes, of course.

**Comment by fhahn** [comment]
run aarch64 tests

**Pull request merged**